### PR TITLE
Ncl5719

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -1,9 +1,138 @@
...
+The following plugin is required:
+
+ * https://plugins.jetbrains.com/plugin/6317-lombok/
+
+It is also recommended to install:
+
+ * https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter
+    * (which allows importing of the code style)
+ * https://plugins.jetbrains.com/plugin/7442-gmavenplus-intellij-plugin
+    * (especially if using the Groovy [example project](https://github.com/project-ncl/manipulator-groovy-examples) )
```
Comment (dwalluck) [inline DEVELOPING.md:76]: [GMavenPlus IntelliJ Plugin](https://plugins.jetbrains.com/plugin/7442-gmavenplus-intellij-plugin) is deprecated and no longer seems available when searching the marketplace.

File: DEVELOPING.md [hunk 2]
```diff
@@ -1,9 +1,138 @@
...
+## IntelliJ
+
+The following plugin is required:
+
+ * https://plugins.jetbrains.com/plugin/6317-lombok/
+
+It is also recommended to install:
+
+ * https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter
+    * (which allows importing of the code style)
```
Comment (dwalluck) [inline DEVELOPING.md:74]: Should we mention or link to [Eclipse Code Formatter](https://github.com/krasa/EclipseCodeFormatter#instructions) on how to set up the code formatter? Unfortunately, I seem to get different results between that and [formatter-maven-plugin](https://github.com/revelc/formatter-maven-plugin).

File: DEVELOPING.md [hunk 3]
```diff
@@ -1,9 +1,138 @@
...
+a HashMap of all properties ; this is used for validation input checking within the CLI /  EXT.
+
+
+## Style & Copyright
+
+Eclipse compatible `codestyle.xml` and `eclipse.importorder` files are supplied inside the `ide-config` directory which
+may also be imported into IntelliJ via the EclipseCodeFormatter. There is also an IntelliJ compatible copyright template
+suitable for use when the project is imported into IntelliJ.
+
+The `.idea` folder contains a copyright template suitable for use when the project is imported into IntelliJ.
```
Comment (dwalluck) [inline DEVELOPING.md:63]: This requires manual setup too? It doesn't just recognize the `.idea` folder automatically, right?

File: DEVELOPING.md (hunk 1 above)
Comment (rnc) [reply DEVELOPING.md:76]: Agreed, this is probably historic as according to https://github.com/mycila/gmavenplus-intellij-plugin/issues/3 it is now included in IntelliJ

File: DEVELOPING.md (hunk 2 above)
Comment (rnc) [reply DEVELOPING.md:74]: I will add a link. Probably investigating `.editorconfig` files might be a good way forward but not relevant I think for this ticket.

File: DEVELOPING.md (hunk 3 above)
Comment (rnc) [reply DEVELOPING.md:63]: I think so.

**Pull request merged**